documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,45 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+With this step you can store one or more key value pairs of data stored in a group called a map or key value map.
+
+To consume the ApiKeyValueMapUpload step, proceed as follows:
+
+* Get the SAP API management service key from the SAP BTP sub account cockpit, under instance and subscriptions for service API Management, API portal under apiportal-apiaccess plan.
```

Comment:
I don't understand this sentence, please rephrase.

Reply:
done